concourse/scripts/entry.sh

```diff
@@ -124,10 +125,15 @@ function setup_gpadmin_bashrc() {
     } >> /home/gpadmin/.bashrc
 }
 
+function install_plpython3() {
+    PGUSER=gpadmin gppkg -i bin_plpython3/*.gppkg
```

Comment:
why we need to add `PGUSER=gpadmin` prefix here?

Reply:
gppkg needs to run sql and by default, it uses the current user which is root. But our gpdb setup doesn't have a root user.